test/player.spec.js

```diff
@@ -0,0 +1,23 @@
+import { Player } from '../src/app/Player';
+
+describe("Player's logic", () => {
+  let player = new Player();
+
+  it('Ask player a question, by calling onQuestionAsked function', () => {
+    let question;
+    let onQuestionAsked = jest.fn();
```

Comment:
Tak samo tutaj wszędzie const.

Reply:
![image](https://user-images.githubusercontent.com/31566345/103242766-c4f05800-4957-11eb-99e8-b446d5dde6f1.png)
przy "question" i "chosenAnswer" musi być let, bo jak się dowiedziałem:
"JavaScript const variables must be assigned a value when they are declared:" stąd błąd .... zabawne ...

Reply:
no chyba, że mam im przypisać stringi jakieś jak wcześniej, ale to wg mnie zbyteczne

Reply:
A no tak. W sumie to jest wlaśnie JS :P Bo question powinno być jakoś zdefiniowane. Ale może jak piszesz - uznajmy to za zbyteczne na razie.